pom.xml

```diff
@@ -141,7 +142,7 @@
             <dependency>
                 <groupId>org.springframework.cloud</groupId>
                 <artifactId>spring-cloud-netflix-core</artifactId>
-                <version>1.0.0.BUILD-SNAPSHOT</version>
+                <version>${spring.cloud.version}</version>
```

Comment:
These don't have the same versions anymore. See [this pom.xml](https://github.com/spring-cloud/spring-cloud-consul/blob/master/pom.xml#L60-L62) in s-c-consul for an example.